pkg/network/node/egressip_test.go

```diff
@@ -72,7 +73,7 @@ type egressOVSChange struct {
 // the new/current set of flows in flows.
 func assertOVSChanges(eip *egressIPWatcher, flows *[]string, changes ...egressOVSChange) error {
 	oldFlows := *flows
-	newFlows, err := eip.oc.ovs.DumpFlows("table=100")
+	newFlows, err := eip.oc.ovs.DumpFlows("table=101")
```

Comment:
instead of hard coding the table numbers could we make a file that stores the table numbers and priorities or whatnot as constants? perhaps pkg/types/constants.go?

Reply:
We could, I would however not do that for the entire openshift-sdn setup, since that would make this PR a lot bigger than what it already is. I could apply that to only the egress IP and egress network policy tables though.

Reply:
I had a branch a very long time ago that let you use "variable names" in OVS flows so you could say stuff like like `"table=${EGRESS-IPS}"` but it ended up not working well, IIRC

Reply:
(https://github.com/openshift/origin/commit/479264578ca5e96c31749e65fbaaeee3a6320721 in https://github.com/openshift/origin/pull/10190)

Reply:
OK, so I don't want to make this PR too intricate this close to code freeze. I am thus going to skip doing this for now. If we feel this really deserves attention going forward, then let's address it during the 4.9 cycle.